src/main/kotlin/github/GitHubImpl.kt

```diff
@@ -136,11 +136,40 @@ object GitHubImpl {
         files: Map<String, String>,
         packageIdentifier: String,
         packageVersion: String,
-        updateState: VersionUpdateState
+        updateState: VersionUpdateState,
+        terminal: Terminal
     ): GHPullRequest {
+        if (
+            data.PreviousManifestData.installerManifest?.equals(data.InstallerManifestData) == true &&
```

Comment:
This is just comparing the object to another object. This isn't comparing the data classes.